docs/data/tree-view/rich-tree-view/selection/ParentChildrenSelectionRelationship.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import { RichTreeView } from '@mui/x-tree-view/RichTreeView';
+import { TreeViewBaseItem } from '@mui/x-tree-view/models';
+import { useTreeViewApiRef } from '@mui/x-tree-view/hooks';
```

Comment:
To avoid the blank line in the JavaScript output?

```suggestion
import { useTreeViewApiRef } from '@mui/x-tree-view/hooks';
import { TreeViewBaseItem } from '@mui/x-tree-view/models';
```

Reply:
#13237